Travel_fellowships.md

```diff
@@ -119,12 +119,9 @@ Any panelist with a clear *conflict of interest*[1] will recuse themselves from
 The selection of the fellowship awardees is based on the criteria described below. 
 We expect applicants to include sufficient information in their application to allow the review panel to assess their eligibility fairly.
 
-***A rubric is currently under development, which will be added to this page upon approval from the OBF board members.***
-
 #### Event and nature of participation
 
 The review panel will assess how the event that the applicants have selected and the nature of their participation promote open science practices in bioinformatics and/or biological research community.
-
 It's important to keep in mind that the event should be scheduled to occur anytime in a one-year window starting one month after the application deadline.
 
 Events that are eligible may include but are not limited to conferences, workshops, code fests, hackathons, training courses, collaborative sprints, informal meet-ups or other skill-building and networking event. 
```

Comment:
Although we fund all these events, the majority still consider the funding to be aimed at attending conferences. I think we can get more applications from skill-building events (workshops, training courses, etc). Let's find ways to highlight the kind of events they can apply to (maybe through a blog-post)...

Reply:
Thank you for adding this in the blog.